pkg/config/polling_manager.go

```diff
@@ -157,21 +152,8 @@ func (cm *PollingProjectConfigManager) SyncConfig(datafile []byte) {
 		return
 	}
 	cmLogger.Debug(fmt.Sprintf("New datafile set with revision: %s. Old revision: %s", projectConfig.GetRevision(), previousRevision))
-	cm.projectConfig = projectConfig
-	if cm.optimizelyConfig != nil {
-		cm.optimizelyConfig = NewOptimizelyConfig(projectConfig)
-	}
 	closeMutex(nil)
-
-	if cm.notificationCenter != nil {
-		projectConfigUpdateNotification := notification.ProjectConfigUpdateNotification{
-			Type:     notification.ProjectConfigUpdate,
-			Revision: cm.projectConfig.GetRevision(),
-		}
-		if err = cm.notificationCenter.Send(notification.ProjectConfigUpdate, projectConfigUpdateNotification); err != nil {
-			cmLogger.Warning("Problem with sending notification")
-		}
-	}
+	_ = cm.setConfig(projectConfig)
```

Comment:
if a function returns an error then we need to check it. It is ok to skip checking in tests, but not in production code.